src/main/java/edu/unc/lib/boxc/migration/cdm/services/MigrationProjectFactory.java

```diff
@@ -21,10 +21,18 @@ public class MigrationProjectFactory {
     private MigrationProjectFactory() {
     }
 
-    public static MigrationProject createMigrationProject(Path path, String name,
-                                                          String collectionId, String user, String cdmEnvId)
+    public static MigrationProject createCdmMigrationProject(Path path, String name,
+                                                             String collectionId, String user,
+                                                             String cdmEnvId, String bxcEnvId)
             throws IOException {
-        return createMigrationProject(path, name, collectionId, user, cdmEnvId, null);
+        return createMigrationProject(path, name, collectionId, user, cdmEnvId, bxcEnvId, "cdm");
+    }
+
+    public static MigrationProject createFilesMigrationProject(Path path, String name,
+                                                          String collectionId, String user,
```

Comment:
We can remove collectionId here since that only applies to CDM projects, and pass null into createMigrationProject